sandbox/App.jsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 
-import { Renderer } from '@prezly/content-renderer-react-js';
+import { Renderer, RendererContextProvider } from '@prezly/content-renderer-react-js';
+import { STORY_BOOKMARK_NODE_TYPE } from '@prezly/slate-types';
 
-import '@prezly/content-renderer-react-js/styles.css'
+import '@prezly/content-renderer-react-js/styles.css';
 import './styles.css';
 
 import story from './story.json';
+import storySample from './storySample.json';
 
 export const App = () => (
     <div className="App">
-        <Renderer nodes={story} />
+        <RendererContextProvider
+            elements={{
+                [STORY_BOOKMARK_NODE_TYPE]: {
+                    getStory: () => storySample,
+                },
+            }}
+        >
+            <Renderer nodes={story} />
+        </RendererContextProvider>
```

Comment:
I think we are able to find a better, more elegant, approach for this. I don't yet have anything specific to suggest, but I will spend time thinking about it today.